packages/angular/tsconfig.json

```diff
@@ -14,7 +14,8 @@
     "module": "es2020",
     "esModuleInterop": true,
     "paths": {},
-    "lib": ["dom", "es2019", "dom.iterable"]
+    "lib": ["dom", "es2019", "dom.iterable"],
+    "types": ["jest"]
```

Comment:
We already include `@types/jest` in the devDeps, why do we need it here?

Reply:
I'll remove the types in this file.

Reply:
b226db3